src/steps/access.ts

```diff
@@ -85,6 +85,7 @@ export async function fetchUsers({
           mobilePhone: employee?.mobilePhone,
           workPhone: employee?.workPhone,
           supervisor: employee?.supervisor,
+          lastLogin: parseTimePropertyValue(user.lastLogin),
```

Comment:
Any unit tests around this?

Reply:
Whoops.  Didn't get my test update pushed.  Should be there now.  Thanks!

I didn't add in any new tests, just updated the existing to add this property.